osu.Game/Utils/FormatUtils.cs

```diff
@@ -7,18 +7,18 @@ public static class FormatUtils
     {
         /// <summary>
         /// Turns the provided accuracy into a percentage with 2 decimal places.
-        /// Omits all decimal places when <paramref name="accuracy"/> equals 1d.
         /// </summary>
         /// <param name="accuracy">The accuracy to be formatted</param>
+        /// <param name="alwaysShowDecimals">Whether to show decimal places if <paramref name="accuracy"/> equals 1d</param>
         /// <returns>formatted accuracy in percentage</returns>
-        public static string FormatAccuracy(this double accuracy) => accuracy == 1 ? "100%" : $"{accuracy:0.00%}";
+        public static string FormatAccuracy(this double accuracy, bool alwaysShowDecimals = false) => accuracy == 1 && !alwaysShowDecimals ? "100%" : $"{accuracy:0.00%}";
 
         /// <summary>
         /// Turns the provided accuracy into a percentage with 2 decimal places.
```

Comment:
Kind of no longer true. I'm overall not too happy about this `alwaysShowDecimals` flag. Why `0.00%` but not `100.00%`? At this point if we're centralising we should centralise, potential discrepancies with web be damned.

Reply:
If I understand correctly, this wasn't true beforehand either - `100%` never had decimals shown before this change, so, if anything, it brings that closer to truth if the flag is set to `true`.

> Why `0.00%` but not `100.00%`? 

Good question to ask the original author of this method.

Reply:
Honestly feels like wasting time going back and forth on significant digits. Should pick an opinionated choice and stick with it.

Reply:
Agreed, I also think we should just stick to 2 decimal places everywhere, as it'ss the most consistent option.